ksqldb-engine/pom.xml

```diff
@@ -197,23 +197,5 @@
                 </configuration>
             </plugin>
         </plugins>
-
```

Comment:
This and the other 3 modules don't have any resource folder and don't need to get the version. Thus, it seems appropriate to just delete those definitions in the pom (might have be added via c&p?)